### DISCUSSION TITLE
🥳 Let's get to know each other!

### DISCUSSION THREAD

**Discussion by daisychaussee** [microsoft/hack-together-dotnet#4] [General]
Hi everyone, we are so excited to kickoff this hackathon on November 20th! Let's use this discussion to introduce ourselves and get to know each other. 🥳

Comment below your name and a bit about yourself! 👇

**Comment by justinyoo** [comment]
Hey, hey, hey, everyone! 👋

It's soooo good to meet you all of you joining our .NET Hack Together. I'm Justin joining this awesome event and helping you to get yourself hack and build nice cloud-native applications with .NET and Azure!

**Reply by evanskiprotich** [reply]
hey, everyone! 👋

It's so nice to meet you all of you joining our .NET Hack Together. I'm Justin joining this awesome event and helping you to get yourself hack and build nice cloud-native applications with .NET and Azure!

**Reply by jwevans** [reply]
Hi Justin, I see you are one of the maintainers and I have a question for you - I've had a read through https://github.com/microsoft/hack-together-dotnet and registered and also had a look at the "Request Access to Azure OpenAI Service" form using this link https://aka.ms/oaiapply. I am interested in taking part but as an individual rather than via a company, is that possible as it seems as though everything is geared towards having a work / schools Microsoft account and the Request Access to Azure OpenAI Service form has a lot of company related questions. Kind regards James.

**Reply by renjithmp** [reply]
Hello Everyone, 
I am Renjith, Based out of India. Very much excited to be part of this event. Hope to see a lot of good ideas coming out here. 
All the best everyone

**Reply by justinyoo** [reply]
@jwevans Oh, thanks for the question! At this time, you've already seen this post: https://github.com/microsoft/hack-together-dotnet/discussions/23 for you to get Azure OpenAI access!

**Reply by Ashahet1** [reply]
@justinyoo I received email for limited access to Azure OpenAI Service, when I registered online which consists of Key and URL, clicking on URL and providing Key, gives me error of invalid event code, What am I missing? Help!

**Reply by aaronpowell** [reply]
@Ashahet1 the event code is a combination of a GUID and your GitHub ID, so it would be in the format of `00000000-0000-0000-0000-000000000000/Ashahet1` that you provide to the playground or the Azure OpenAI SDK.

**Reply by chanuri** [reply]
Hi All, I'm Chanuri from New Zealand. I'm really excited to be part of this. :)

**Reply by Ashahet1** [reply]
@aaronpowell. I tried as advised, but still have same error, has invalid key error.

**Reply by aaronpowell** [reply]
@Ashahet1 I've looked up your access token and tried it on the playground, and verified that it works via the playground. Is the problem you are seeing in the playground or via the Azure OpenAI SDK?

**Reply by Ashahet1** [reply]
@aaronpowell . I meant playground and now key can be authorized, Thanks for help!

**Reply by Ashahet1** [reply]
@aaronpowell Also, How to access Open AI SDK, Filled out form but never received email, or I am missing something?

**Reply by justinyoo** [reply]
@Ashahet1 It seems that you've already received the GUID as an access code. Then, use that access code by following this document: https://github.com/microsoft/hack-together-dotnet/blob/main/OPENAI-README.md#how-to-use-the-proxy

Basically, we provided you with the endpoint and access code that mimics the **real** Azure OpenAI endpoint and API key.

**Reply by Nirajparmar68** [reply]
Hello Everyone!
I think I am too late here lol
I am a dot net developer working at IT Path Solutions in Ahmedabad, India.
Having 4+ years of experiance and passionate about programming.

**Comment by aaronpowell** [comment]
G'day folks,

I'm Aaron and I'm located in Sydney, Australia 🦘. I'm here to help you all out with building .NET applications using AI and I can't wait to see what you come up with!

**Comment by ambanks** [comment]
Hey everyone!

I'm Angel. We're happy you joined The Great .NET 8 Hack and can't wait to see the projects you come up with. Please let us know if you have any questions.

**Comment by CitruzCo** [comment]
Hi, 
I am trying to request access to Azure Open AI , but it says form is closed. Any idea?
 https://customervoice.microsoft.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR7en2Ais5pxKtso_Pz4b1_xUOFA5Qk1UWDRBMjg0WFhPMkIzTzhKQ1dWNyQlQCN0PWcu&culture=en-us&country=us

**Reply by aaronpowell** [reply]
Looks like they've changed the form link, I've created #13 to fix the README, but the correct link is https://aka.ms/oaiapply

**Reply by CitruzCo** [reply]
Thanks!

**Comment by PeDrumond** [comment]
My name is Pedro, I'm from Brazil!!! Let's code together!!!

**Comment by NavneetHegde** [comment]
Hey!
Navneet here, excited to join The Great .NET 8 Hack

**Comment by davidbuckleyni** [comment]
Hi All looking forward to this will be good to be a part if a can

**Comment by affiahmedkhan** [comment]
Excited to get started with the .Net 8 amazing code journey with #HackTogether hackaton, am Aftab Ahmed from Pakistan

**Comment by mheidari988** [comment]
Hello everyone,
I'm Reza, and I'm thrilled to join and work alongside anyone who shares the same passion.

#HackTogetherDotNet

**Comment by ankurmay1988** [comment]
Hello everyone,
I'm Ankur from India, Excited for .NET8 ! Can't wait to hack together !

**Comment by gerardo-lijs** [comment]
Hi, looking forward for my first hackaton with .NET and ML! Hopefully I can tag along a team with more experience in hackatons and hack together some interesting project.

**Comment by prdLaturMH24** [comment]
Hello Everyone,
I'm Pramod from India, Excited for .NET 8! Looking forward to my first hackathon with .NET and wish to join and work in a team.

**Comment by kirisky** [comment]
Hi! Hope to code with you guys together! 😄

**Comment by allalakhayri** [comment]
Hi guys , this is my first time developing using .NET technology , it will be a good chance to know fellows , let's get in touch .
Happy coding !

**Comment by CheeloHamududu** [comment]
Hi Everyone, excited for this Hackathon.
My name is Cheelo and i'm from Zambia, i look forward to developing and learning from everyone here.
This will be the first time using .NET

**Comment by Pradeepkishan1** [comment]
Hey, Pradeep Kishan here so excited to be part of NET8  Can't wait to code together!

**Comment by Megasoft2** [comment]
Hello everyone. I am so excited not to win a prize but just to learn.

Greetings
Jorge
Country Venezuela

**Comment by dejandjenicaspen** [comment]
Good luck everyone

you will need it!!!

**Comment by hichi-dev** [comment]
Hi everyone, 
My name is Hicham. I am looking forward to learning more about .Net 8 
Greetings from Zurich

**Comment by MaggieMikhael5** [comment]
Hello Everyone,

My Name is Maggie from Egypt. I'm looking forward to learning .NET.

**Comment by citsuresh** [comment]
Hello Everyone! I am Suresh from India. I am very excited to join this hackathon. I have good experience developing .NET applications, primarily Winforms, and WPF. I would like to learn .NET8  and develop exciting apps.

**Comment by eincioch** [comment]
👋Hello everyone! My name is Enrique Incio and I am from Peru-Trujillo. I am very excited to join this hackathon. I have experience in developing .NET applications, mainly backend (API) and Mobile (Xamarin). I would like to learn a lot more about you. NET8 and develop interesting applications, with Azure OpenAI.

🗣️ website: https://devenriqueincio.web.app/
🎯 WhatsApp channel: https://whatsapp.com/channel/0029Va9RY0SFMqrRZXcWRd1I
📢 github: https://github.com/sponsors/eincioch

See you on November 20th!! 💪

**Reply by LucioD3v** [reply]
Que tal Enrique, te gustaría formar equipo?

**Comment by YeinsM** [comment]
Hello everyone this is Wilder Mancera an enthusiastic developer, who live in Dominican Republic, but I'm from Colombia, I hope together learn as possible to this experience. A .NET lover, 2 experience years working as a developer

**Comment by shreenivaska** [comment]
Hi!, Shreenivas here. I want to learn AI and GenAI. Hope to win some dough! ;)

**Comment by satsvelke** [comment]
Hello everyone, I am sats from India, very excited for .net 8, lets code together

**Comment by tarangpandya** [comment]
Hey folks, so good to see you all.. very much excited.. lets code together.

**Comment by SubirDas08** [comment]
Hello @Everyone,
I am Subir from Sweden. I'm eager to start using AI to build .NET apps using .NET 8, and I look forward to seeing what you come up with!
✌️✌️✌️

**Reply by Ashahet1** [reply]
Would you like to team up or Fly solo? I am too interested in building .NET apps using AI or similar idea..✌✌✌✌

**Comment by GeeHill** [comment]
Hello Everyone!

My name is G. What a pleasure it is to welcome each and every one of you to our.NET Hack Together. I'm thrilled to be a part of this fantastic gathering where we embark on a journey to craft sophisticated, cloud-native applications using the powerful duo of.NET and Azure. Let's dive into the world of coding brilliance together!

**Comment by Stein02** [comment]
hello developers,
am stein, here is an opportunity for us to learn and show what we have.....all the best everyone.

**Comment by ScienceBeyondTime** [comment]
🎶😎⚡ I'm Lorrie from Adelaide... Sounds like a great excuse to test our local network infrastructure 😂😂

**Comment by ekremersahin** [comment]
Hello everyone, I'm Ekrem from Turkey. I'm excited for this hack.

**Comment by Ashahet1** [comment]
Hello Folks!

I am AJ. So Excited to be part of Hack Week and team up to build a project with people who are interested in building up AI project using .NET 8.

**Comment by ramarbstr12play** [comment]
Hi everyone my name is Ahmed from Florida and I am excited to be part of this event!

**Comment by syedfahadiqbal7** [comment]
👋Hello everyone! My name is Syed Fahad Iqbal and I am from Karachi, Pakistan. I am very excited to join this hackathon. I have experience in developing .NET applications, Desktop/Web/ Backend API/ FrontEnd. I would like to learn a lot more about you. NET8 and develop interesting applications.

**Comment by MBirajdar** [comment]
Hello Everyone , I am Manoj Birajdar from India. I am very excited to join this hackathon. I would like to learn a lot more about .NET and develop interesting applications.

**Comment by Kroepniek** [comment]
Hey everyone,

I'm Daniel and I'm from The Netherlands 🧀🚲. Excited to participate in the project.

**Comment by AnnaGolosova** [comment]
Hey coders! 👋 

I'm Hanna, your .NET Cloud developer. Ready to dive into the coding adventure together? 

Let's rock this hackathon! 🚀

**Comment by Dkaewae** [comment]
hey everyone am Douglas from Papua New Guinea, am excited to be part of this coding. Am new to this environment and am looking forward to meet some of the participants to be part of the team and to learn more.

**Comment by jbp17** [comment]
Hi everyone, I am Jing from the US.  I am excited to join the hackathon and learn more about the .NET8.

**Comment by Dv1101** [comment]
Hello folks, let's rock on 20th 😎💪🏻

**Comment by cmasdev** [comment]
Hi everyone. Nice to be here. Let's code with .NET 8 here!

**Comment by gowthamece** [comment]
Hello everyone, I'm Gowtham. I'm thrilled and excited to join this hackathon.

**Comment by vishalbhatd123456** [comment]
Hello everyone, Vishal here from Bangalore, India. Excited to learn and explore .NET8 and the hackathon, atb to all!

**Comment by JayawardhanaJTH** [comment]
Hello my friends,

I'm Tiran from Sri Lanka. I love to learn new things and coding. Very excited to get experience in .NET 8. 
Happy coding journey friends!

**Comment by AdityaSkhorne** [comment]
Hello everyone,
My name is Aditya Shankar Khorne
I am a second year computer science student currently pursuing my BE degree
from an Indian University.
I love to write programs and learn with like minded people

**Comment by kickstart530** [comment]
Hi Everyone!,

I am here to showcase a solution that would help me build on my experience and work with others to collaborate here. 

Thank you.

**Comment by patrick-lim** [comment]
Hi everyone,

I'm Patrick from Canada. Looking forward to coding.

Thanks

**Comment by JaneConan** [comment]
Hello everyone👋

I'm Conan Yao from TianJin, China. 

What a pleasure to have a friend come from afar! 

Looking forward to coding💖

Thanks all ~

**Comment by ajmalrasouli** [comment]
Hi everyone,
I am Ajmal from London.

I am excited to been part of this Hackathon.

Thank you!

**Comment by piotr-andrzejewski** [comment]
Hello,

my name is Peter and I'm excited to join The Great .NET 8 Hack :)

**Comment by govindagoud-patil** [comment]
Hi everyone, I am Govind from the US. I am excited to join the hackathon and learn more about the .NET8.

**Comment by codemillmatt** [comment]
Hi everyone! My name is Matt and I'm from Seattle! ☂️🌧️ I'm helping run the Hack Together and would love to answer any questions you may have - please reach out! I'm looking forward to seeing what you all build!

**Reply by jwevans** [reply]
Hi Matt, I have a question for you - do I need a work / schools Microsoft account to take part or should I be able to use my personal hotmail.co.uk account. Also the "Request Access to Azure OpenAI Service" (https://aka.ms/oaiapply) has a lot of company related questions and gives the impression you can only take part if you have a Microsoft work / schools account but this is a personal leaning objective not related to a company. Kind regards James.

**Reply by codemillmatt** [reply]
Hi James - good news, we worked out a way to get proxy access to Azure OpenAI without going through the full application process. Use this https://aka.ms/hacktogether/dotnet/AzureOpenAIAccess to get on the list. It’ll open up on Nov 20.

**Reply by jwevans** [reply]
Hi Matt, That's great news, thank you - I have completed that registration form using my hotmail.co.uk account. Looking forward to the event, sounds like a lot of fun!

**Comment by fazlarabbi2** [comment]
Hello Everyone, **Fozle Rabbi** here from **Bangladesh**. I am beginners just learning now C# advanced topics and excited to join the hackathon and learn more about the .NET 8.

**Comment by Shak-o** [comment]
Hello everyone, I'm Shako from Georgia, and I'm thrilled to be a part of the .NET 8 hackathon. Thank you for organizing it. Wishing everyone good luck!

**Comment by savemejeff** [comment]
Hi everyone, I'm jeff, a junior software developer from China. It's my first time joining a hackathon, good luck for me :)

**Comment by omarwasfi** [comment]
Hello, 
I'm omar wasfi. Software engineer 💻
Who loves : New experiences & new Friendships🤞
Skate - Run - Travel 🌍
All of that with Coffee ☕ & nice music 🎶 of course
excited to get to know you and to join this hackathon.

**Comment by AThomasBring** [comment]
Hi everyone! Anton here from the UK. Looking forward to learning more about .Net 8!

**Comment by tajbender** [comment]
Hi everyone,

I'm Thorsten from Germany.

Glad I found my way to this Hackathon to learn more about the surprises .net 8 has to offer. And finally, what will happen here the next weeks.

Happy coding to everyone.

Regards,
tajbender

**Comment by romeyng** [comment]
Hi All!
I'm Romeyn from a little island in the Caribbean called St.Vincent. I'm glad to be a part of this event and looking forward to learning from everyone!

**Comment by Sidzeppelin95** [comment]
Hello Hackers,
My name is Siddhartha Sharma from India. I am excited for Open API and Open AI, hope this hackathon rocks!!

**Comment by MichaelZhouNz** [comment]
Hello everyone. I'm Michael from New Zealand. I am keen to learn more about .NET 8!

**Comment by shahidkhanafridi** [comment]
Hello everyone,

**Muhammad Shahid Khan Afridi** from **Pakistan**, I have a long journey of coding with day-to-day versions of the DotNET ecosystem, but this first time going with the worldwide DotNet community in this hackathon. I am hoping for knowledge and new things from the community and also want to share the knowledge I gained in past years.

Thank you, everyone,

**Comment by EdwardMoon85** [comment]
Hello guys
I am Edward, lived in Rep.Korea. I am very interested in coding and I am very excited about the new .NET. I will attend this event with great pleasure.

**Comment by Vivekkosare** [comment]
Hi everyone,
I'm Vivek from Stockholm, Sweden. I'm happy to join this hackathon and would love to learn more about it.

**Reply by vithal-wadje** [reply]
Good to See you @Vivekkosare , Best wishes.

**Reply by Vivekkosare** [reply]
Nice to see you too @vithal-wadje 👍 :). Thank you. I hope you are doing well.

**Comment by danirzv** [comment]
Hi!
I'm Danial from Iran, I've heard about this event 5 mins ago 😅
Lets have fun!

**Comment by Kahbazi** [comment]
Hey folks!
I'm Arvin. I've been using .NET for a while now and I'm happy to participate in this event.

**Comment by Hinthusiast** [comment]
Hello everyone! I'm a tech hobbyist career shifter from the Philippines and glad to be part of this awesome event! Thank you for accepting my request for the limited access to Azure OpenAI. Let's get it on!😁

**Comment by vithal-wadje** [comment]
Hi Everyone,

I am Vithal from India. I have been working with the Microsoft technology stack for around 10 years. I am a technology enthusiast and an active blogger, excited to participate in this event.

**Comment by VasanthRajan2** [comment]
Hello Everyone,
I am Vasanth from Canada. I am working on .NET since 2004. Excited to see new features and roadmap for .NET 8. Happy coding!!!

**Comment by florianguigue** [comment]
Hello World !
I'm Florian from France. Pretty new to .NET but senior developer in Java since 2016. Can't wait to see the resulting projects of this hackathon

**Comment by harshalschaudhari** [comment]
Hi,
I am Harshal from India. I have been working on .NET since 2007. Excited to see the new features of .NET 8 !!!

**Comment by ghost** [comment]
Hello! I'm Artem form Ukraine. I recently started working with .NET. I am glad to take part in the hackathon and learn new knowledge from this event. Good luck for everyone!

**Comment by ghost** [comment]
Greetings guys,
I hope you'll doing great. I'm university student in computer science majors and like to participate in the contribution for .NET 8 **[Hack](https://github.com/microsoft/hack-together-dotnet)**. Would you like to team up ?
 I'll see you there at [live](https://www.youtube.com/watch?v=MPzNDEBCq4M) stream.

Thanks,
*-* ajay

**Reply by tajbender** [reply]
Hi, @stlajay,
Any idea what to do as a team?
Regards,
tajbender

**Comment by rjwilsondev** [comment]
Hi! I'm Ryan, .NET / Azure / JS  enthusiast. Looking forward to digging into Aspire, ASP.NET, and all the goodies that dropped in .NET 8!

**Comment by Tuck1297** [comment]
Hi everyone! My name in Tucker and I'm hacking from Minnesota! I know C but have never used any version of .NET or participated in any hackaton before. Any suggestions and or tips would be appreciated! I look forward to interacting with some of you and see what the results of this event! Happy Hacking!!!

**Comment by boku-engr0** [comment]
Hi all! This is **Nayem**
Exploring .NET 8 and all the good stuff...I'm so glad to be here ✨
Let's hack together 🔥
You can connect with me on [linkedin](https://linkedin.com/in/md-jannatul-nayem-161858284/)

**Comment by heftif** [comment]
Hi there, this is Fiona. Excited for this hack together event :)

**Comment by EddyBautista-93** [comment]
Hey everyone! My name is Eddy, looking forward to seeing the amazing things we create!

**Comment by Philip-Wong** [comment]
Hi everyone,
Happy to learn about .NET 8. I cannot wait to start a new .NET 8 project and give it a try.

**Comment by brunoalfred** [comment]
Hi Everyone,
I'm new to .NET but I would love to try building something 🤪 with it...

**Comment by dannymiracle** [comment]
Hi all, my name is Danny Miracle. Look forward to creating/solving with you all.

**Comment by LucioD3v** [comment]
Hi everyone,

I am Vicente from México and I'm so happy to learn about .NET 8

**Comment by Yusadem131** [comment]
Hi everyone, I am Abdulakeem Yusuf from Nigeria, I am currently looking for a team member.

**Comment by kingsznhone** [comment]
Hi every one, I'm Alex, excited to join hack together event. I‘m happy to try .NET 8 new features.

**Comment by diegoquinonez1** [comment]
Hello everyone,

I'm happy to join #HackTogether! My name is Diego QUIñonez and I am from Colombia.

**Comment by keshavsingh3197** [comment]
Hi,
I am`Keshav Singh`, a software dev with passion for .Net, Cloud(AWS, Azure), Microservice.
Super pumped to meet you all and see what awesome projects we can build together.

Can't wait get started! 🚀

**Comment by 2652660** [comment]
I am Christopher Michael Don Rogers.

I like to do things people say are impossible.
I like to debate or argue...whichever.
I like to challenge myself slowly and egotistically.

I am a bit odd, I spent a long time not socializing - and was kinda raised by the internet... o.o

**Comment by jeffgoo** [comment]
Greetings .Netizens.  I'm [Jeff](https://www.linkedin.com/in/jeffgoo/) from Seattle and have enjoyed building API's using C#, .Net, MSSQL, and Redis for years.  I can't wait to go cloud native quickly with Aspire and I'm looking forward to building something cool with y'all using .Net 8 and the GPT-3 LLM.

**Comment by ShohruhRed** [comment]
Hi everyone, this is my first time participating in hackathon, I hope to learn something new and make a good project

**Comment by IgorNMS** [comment]
Hullo friends!
I'm Igor, from Brazil!
Very excited to be part of hack-together!!!

**Comment by apedzko** [comment]
Hi All! 

Looking forward to get my hands dirty and start coding :-).

Alex.

**Comment by rahulbcsit** [comment]
Hi All, look forward for .NET8

**Comment by jcr010910** [comment]
I didn't get the email from DoNotReply@hacktogether.net yet. what shall be the next step?

**Reply by Ashahet1** [reply]
You used company email and not gmail correct, I got email in few min. But Key in email doesn't work. 😬

**Reply by justinyoo** [reply]
@jcr010910 Would you please be able to check your spam mail folder? If you can't find it, please submit another request with another email address. Please make sure that there shouldn't be typo or non-visible space characters.

**Comment by christophebrucker** [comment]
Hi everyone, my name is Chris Brucker and I am excited to join this event. I am working in healthcare informatics and eager to build an awesome project !

**Comment by kronnos1992** [comment]
Hello! every one there, i am new in .net world and i am exciting to grow alot with this summit!!

**Comment by Yusadem131** [comment]
@daisychaussee Please, can I use my student-ambassadorship email to apply? Thanks.

**Reply by justinyoo** [reply]
@Yusadem131 As long as it's a valid email address that we can reach out, it should be fine.

**Comment by Yusadem131** [comment]
@daisychaussee Please, can we get the recordings of the last sessions. Thanks

**Reply by aaronpowell** [reply]
The videos are on the Visual Studio Twitch channel. [Here's the keynote](https://www.twitch.tv/videos/1982894061) and [here's the generative AI session](https://www.twitch.tv/videos/1984371351).

**Reply by codemillmatt** [reply]
Heya @Yusadem131 - you can also see them on the .NET YouTube channel. https://www.youtube.com/@dotnet/streams and you can register there to get reminded of future streams!

**Comment by abimr** [comment]
Hello from Denmark 😊

Are we allowed to submit a project that uses ASP.NET Core on the backend and React/Typescript as the frontend? Or do you prefer .Net only projects?

**Reply by aaronpowell** [reply]
As long as the core functionality of the app is in .NET, you can mix other technologies in.

**Comment by DarkBrain-LP** [comment]
Hi everyone, I am Yawo Yovo, computer engineering student at the UTMB (France).
I'm so excited to participate to this #hack

**Comment by vende6** [comment]
Hello, my name is Damir and I come from Bosnia and Herzegovina. Such an incredible opportunity to participate and certainly looking forward to hack this together with all of you!

**Comment by Surya056** [comment]
Hello everyone, 
I'm Surya from India. I'm excited for .NET8 and can't wait to start coding!

**Comment by Sidzeppelin95** [comment]
Hi Everyone,
Do you guys know when the coding and project build will start?

**Reply by tajbender** [reply]
https://github.com/microsoft/hack-together-dotnet

Regards

**Comment by Sidzeppelin95** [comment]
I have received the proxy link but the event code does not work. Is it the
same as the key provided for the workshop? It worked initially where I was
able to login to the dashboard but it's not working now. Please suggest!

**Reply by aaronpowell** [reply]
I've added some more instructions on using the proxy here: https://github.com/microsoft/hack-together-dotnet/discussions/23#discussioncomment-7668396

**Comment by shreenivaska** [comment]
Hi, https://aoai.hacktogether.net/ is not accepting my key. What to do?

**Reply by aaronpowell** [reply]
Please refer to the connection information here: https://github.com/microsoft/hack-together-dotnet/discussions/23#discussioncomment-7668396

**Comment by tinycrops** [comment]
Novelty! Competition! Excitement! Difficulty following instructions!

**Comment by aaronpowell** [comment]
The SDK is a public NuGet package https://www.nuget.org/packages/Azure.AI.OpenAI/1.0.0-beta.9

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by Ashahet1** [reply]
I tried using SDK, but still ask me to Request access 
![image](https://github.com/microsoft/hack-together-dotnet/assets/39419655/1d77a874-fa06-4380-aaa1-596fd2cf0af5)

**Reply by justinyoo** [reply]
@Ashahet1 Please use the playground page that we provide. Your access code doesn't directly work on Azure OpenAI Studio. If you want to use the **real** Azure OpenAI Studio, please apply this form: https://aka.ms/oaiapply

**Reply by codemillmatt** [reply]
And just so you have it, the playground can be accessed at: https://aoai.hacktogether.net

**Comment by zawhtut** [comment]
Hi @daisychaussee 
I would like to check whether I registered for this event properly. I didn't receive the confirmation email.
Thanks.

**Reply by ambanks** [reply]
Hi @zawhtut. You wouldn't have received a confirmation email but you should receive emails from periodically with updates/announcements.

**Comment by SomebodyToLove1337** [comment]
Hello everyone,

my name is Benedikt and I have been working as a system administrator for about 20 years. However, I have also been interested in programming for a long time and have done and learned a lot in PowerShell. 
At the beginning of this year I found the HackTogether events from Microsoft and immediately fell in love.
Now I have already participated in the HackTogether events Microsoft Graph, Microsoft Teams, Microsoft PowerPlatform. 
In the meantime I have even found a new job as a SharePoint and PowerApps developer where I can start next year. And that's only thanks to the Microsoft HackTogether events and a bit of my own initiative.
Unfortunately, I'm not yet a pro like many here.

But I hope it's okay to deliver a smaller program? I'm not aiming for the main prize, but every badge you can earn just makes me happy.

I'm a bit late now, but I'll make up for it with more work.

My idea would be to create a small textadvanture with AI generated texts, I would use this project as a code base (https://learn.microsoft.com/en-us/dotnet/orleans/tutorials-and-samples/adventure) if I can get it to run on .NET 8.
Do you think the project would be ok?

I wish every participant a lot of fun at the event :)

**Reply by justinyoo** [reply]
@SomebodyToLove1337 Thanks for your interests in our Hack Together events over the year! No matter it's big or small, you are always welcome to submit the app, as long as it's written in .NET 8!

**Comment by AdityaSkhorne** [comment]
Ok

**Comment by perplexedyawdie** [comment]
Hey all! Javel here. Excited to experiment with .Net! 🧙‍♂️

**Comment by OlksBndz** [comment]
Hi Guys~ We are very excited to participate in Hack togethe We are working on an AI co-pilot enterprise project, designed to handle large volumes of documents with varying roles and permissions. This system is tailored specifically for enterprise use. It aims to streamline the handling and management of documentation, ensuring efficient access control and role management. This AI co-pilot will provide a convenient and efficient way to manage enterprise-level documentation, making it an invaluable tool for businesses of all sizes.


Our Team:
Yurii Morozyk is the project manager and developer, 
Khrystyna Ozarko is a .NET developer, focusing on the identity part of the project 
Oleksandr Bendziak is responsible for Business Analysis (BA) and Quality Assurance (QA).

**Comment by shahidkhanafridi** [comment]
Hi everyone, how can I take part in this Hack Together, what are the prerequisites?
And how to start, where I get the project description to work on?

**Reply by ambanks** [reply]
Hi @shahidkhanafridi! Happy you want to join the Hack! Please take a look at the [README.](https://github.com/microsoft/hack-together-dotnet#readme) It has registration information, prerequisites, and info to get started.

**Reply by codemillmatt** [reply]
Hi @shahidkhanafridi - it's up to you on what you want to hack on. Any projects welcome. Make sure you register for the hack: https://aka.ms/hacktogether/dotnet/register and also checkout the overall home page https://aka.ms/hacktogether/dotnet for more instructions.

For the most part - use .NET 8 to create an application that deals with AI or cloud native (or both)! You'll have the opportunity to win prizes if your application is judged to be the best. (And here's a blog announcing everything as well: https://devblogs.microsoft.com/dotnet/join-us-for-the-great-dotnet-8-hack/)

**Comment by usama1947** [comment]
Hello friends, my name is Usama, I am from Pakistan. I am currently a software engineering student. I will be building an immersive and intuitive .NET learning platform.

**Reply by Yusadem131** [reply]
Can we team up together @usama1947 ?

**Comment by edilma** [comment]
Hello Everyone, greeting from Florida (USA).   Happy to be participating the hack together.

**Comment by DerickIT** [comment]
Hey!
Derick here, excited to join The Great .NET 8 Hack

**Comment by soyfrien** [comment]
Hello, I've never finished a hackathon before. But now that I have such a wonderful teacher in Copilot, I just might this time.

I have a question: Should the AI category submissions use OpenAI's technology, or can it also be with Cognitive Services?

**Reply by codemillmatt** [reply]
Hi @soyfrien - you're more than welcome to use the Cognitive Services technologies if you like!

**Reply by soyfrien** [reply]
Excellent, thank you.

**Comment by SteveStrong** [comment]
I am looking for the simplest example to get started,  maybe a console app in .net 8   ,  Can you point me towards that code?

**Reply by codemillmatt** [reply]
Hey @SteveStrong - check out the resources here:  https://learn.microsoft.com/en-us/training/dotnet/

You’ll find docs, videos, and Learn modules that have all recently been updated for .NET 8.

You also could file->new a console app from Visual Studio for a bare bones template.

**Reply by SteveStrong** [reply]
thanks , that is helpful,  however I can looking for a simple console app that called the OpenAI chatgpt  API,   many of the hack examples contain framework code that 'hide' the flow from APIKey to api.

**Reply by codemillmatt** [reply]
Try following the instructions here: https://github.com/microsoft/hack-together-dotnet/blob/main/OPENAI-README.md#how-to-use-the-proxy

Using that inside a normal file=>new console app should get you started.

**Comment by Pablo10544** [comment]
I am Pablo from brazil, Let's code!

**Comment by dljones555** [comment]
Hello, 

Apologies for introducing myself late into the hackathon.

Have dialoged with ChatGPT, BingChat, etc. over the past few months to get a sense of how they answer. Amazing in many ways, but leads to a lot of questions about how things work or how life may change with AI. 

I'm interested in learning more about how to test AI on given topics to gain confidence in using it practically to precision prompt engineering to understanding if AI is used in real-world applications how much human smarts and supervision is required and at what skill level and labor cost.

I'm only a little bit into exploring the AI and the coding side as well. Have a lot of questions and see there is more to explore with all the great research, tools, and SDK's that is out there.

For this effort, have tried a few things with Roger Pincombe's OpenAI nuget package, and just got to exploring the Azure OpenAI API with the contest's offering--thanks to Microsoft for making this available.

David
writing this in Arizona near the Grand Canyon atm

**Comment by dhornet87** [comment]
Hello everyone, 

looking forward to submit my solution to a hackathon.

Malte

**Comment by bprussell** [comment]
Hello everyone. This event looks awesome, but I am just now finding out about it on the last day. For those who participated, I'm curious how you initially found out about the event. I discovered it through a link on MS Learn. I would like to participate in a similar event in the future, so I would like to make sure I'm looking in the right places.

**Comment by SiddyHub** [comment]
Hi All, I have submitted my Application for Hackathon on Github, but I have not received any badge. Can anyone assist me on how to get the same?.

Thanks In Advance, and happy coding everyone :)..

**Reply by boku-engr0** [reply]
Same here ✋

**Reply by annuk123** [reply]
Can you tell me that for which hackathon are you talking about?

**Reply by boku-engr0** [reply]
@annuk123 I believe he was talking abot this one 🙄
![image](https://github.com/microsoft/hack-together-dotnet/assets/64035356/6c0a40db-8601-471e-ac60-f64f7920a777)

**Reply by annuk123** [reply]
Oh, 🙄 Have you participated in this hackathon??

**Reply by boku-engr0** [reply]
@annuk123 you can find it here https://github.com/microsoft/hack-together-dotnet/issues/43

**Reply by codemillmatt** [reply]
Badges will be on their way soon to everybody who submitted a project by the deadline.

**Reply by boku-engr0** [reply]
Glad to know @codemillmatt....Thank you ♥️

**Comment by Sidzeppelin95** [comment]
That's impressive!

**Comment by SmartDeys** [comment]
Hello , @justinyoo 

Due to the recent cyclone and floods in Chennai from the 4th to the 6th, our region experienced severe disruptions, including the absence of power and network connectivity for the entire duration. Regrettably, these challenging circumstances prevented us from creating a demo video for our project during this period. We sincerely appreciate your understanding and kindly request your consideration given the lack of essential resources. Additionally, given the challenges faced, we seek your support in utilizing just one Azure access key, at least for the next day or half-day, as we work to restore normalcy. Your understanding during this time is immensely valued.

**Comment by syedfahadiqbal7** [comment]
Hi Everyone,

Regrettably, it appears that I've missed the deadline to submit any
solutions. Unfortunately, the constraints of my work schedule prevented me
from submitting my application for the GitHub Hackathon in a timely manner.

Thanks & Regards,

*Syed Fahad Iqbal*

*Senior Software Engineer*

+971 55 178 3398 <+971%2056%20935%200190> (Mobile)
syed-fahad-iqbal(Skype)

**Comment by ajmalrasouli** [comment]
Me too still waiting for my badge 📛

Kind regards,

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by codemillmatt** [reply]
Badges should be on their way soon to everybody who submitted a project!

**Comment by ajmalrasouli** [comment]
Here too

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ajmalrasouli** [comment]
Thanks.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
